src/main/java/com/example/demo/controller/SurveyController.java

```diff
@@ -50,5 +52,16 @@ public Survey closeSurvey(@PathVariable int id) {
 		survey.setClosed(true);
 		return repository.save(survey);
 	}
+
+	@PostMapping("/survey/{id}/publish")
+	public ResponseEntity<Survey> publishSurvey(@PathVariable int id) {
```

Comment:
Why does this return a survey?

Reply:
Client-side might want to use this info to update